src/renderer/components/MapFilter/utils/strings.js

```diff
@@ -31,6 +31,7 @@ export function primitiveToString (
   if (typeof value === 'number') return value.toString()
   // TODO: what to show (translated) for "null" field (vs. undefined)
   if (value === null) return 'No Value'
+  if (value === undefined) return undefined
```

Comment:
This will break things, since this function must return a string. However, `value` is type-checked to always be `string | boolean | number | null`.

Reply:
This is what was causing the error though, Autocomplete can accept undefined as a value empty string will show up as an empty string. So we can check it downstream instead

Reply:
Or should it be able to return undefined?

Reply:
Just completed a big overhaul of this code, since the type checking needed to be much more thorough not just on select fields. Pushing now.

Reply:
Nevermind, removed this. Although this function probably shouldn't return empty string for undefined